lib/hu_board/account_helpers.rb

```diff
@@ -25,6 +29,15 @@ def subscription_active?(customer)
       false
     end
 
+    def subscription_canceled?(customer)
+      cus = customer[:stripe][:customer]
+      if cus[:subscriptions][:total_count] > 0
+        sub = cus[:subscriptions][:data][0]
+        return sub[:status] == "canceled"
```

Comment:
Confirming "cancelled" vs "canceled" here, as I've seen related bugs in the past.